tabbycat/notifications/models.py

```diff
@@ -110,25 +110,24 @@ class EmailStatus(models.Model):
     EVENT_TYPE_ASM_UNSUBSCRIBED = 'group_unsubscribe'
     EVENT_TYPE_ASM_RESUBSCRIBED = 'group_resubscribe'
 
-    EVENT_TYPE_CHOICES = (
-        (EVENT_TYPE_PROCESSED, _("Processed")),
-        (EVENT_TYPE_DROPPED, _("Dropped")),
-        (EVENT_TYPE_DEFERRED, _("Deferred")),
-        (EVENT_TYPE_DELIVERED, _("Delivered")),
-        (EVENT_TYPE_BOUNCED, _("Bounced")),
-        (EVENT_TYPE_OPENED, _("Opened")),
-        (EVENT_TYPE_CLICKED, _("Clicked")),
-        (EVENT_TYPE_UNSUBSCRIBED, _("Unsubscribed")),
-        (EVENT_TYPE_SPAM, _("Marked as spam")),
-        (EVENT_TYPE_ASM_UNSUBSCRIBED, _("Unsubscribed from group")),
-        (EVENT_TYPE_ASM_RESUBSCRIBED, _("Resubscribed to group")),
-    )
+    class EventType(models.TextChoices):
```

Comment:
Why'd you keep the `EVENT_TYPE_` attributes above? Not needed anymore with this `enum`.